scripts/rook.sh

```diff
@@ -116,6 +117,22 @@ function delete_block_pool() {
 	rm -f "./newpool.yaml"
 }
 
+function create-block-ec-pool() {
```

Comment:
function names may not contain `-`, use `_` here

Reply:
Updated :+1: